homeassistant/components/nissan_leaf/__init__.py

```diff
@@ -111,8 +114,10 @@ async def async_handle_update(service):
 
     async def async_handle_start_charge(service):
         """Handle service to start charging."""
-        # It would be better if this was changed to use nickname, or
-        # an entity name rather than a vin.
+        _LOGGER.warning(
+            "The 'nissan_leaf.update' service is deprecated and has been replaced by a"
```

Comment:
I think you inverted the warnings.
```suggestion
            "The 'nissan_leaf.start_charge' service is deprecated and has been"
```